Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_test.py

```diff
@@ -306,3 +307,248 @@ def test_list_drives_in_site(command, args, response, expected_result, mocker):
+                                                                  'path': '/drive/root:/test-folder'},
+                                              'file': {'mimeType': 'image/jpeg',
+                                                       'hashes': {'quickXorHash': 'quickXorHash'}},
+                                              'fileSystemInfo': {'createdDateTime': '2022-12-15T12:56:27Z',
+                                                                 'lastModifiedDateTime': '2022-12-28T11:38:55Z'},
+                                              'image': {},
+                                              'shared': {'scope': 'users'}}
+
+return_context = {
+    "MsGraphFiles.UploadedFiles(val.ID === obj.ID)": {
+        "OdataContext": "https://graph.microsoft.com/v1.0/$metadata#sites(some_site)/drive/items/$entity",
+        "DownloadUrl": "some_url",
+        "CreatedDateTime": "2022-12-15T12:56:27Z",
+        "LastModifiedDateTime": "2022-12-28T11:38:55Z",
+        "Name": "some_pdf.pdf",
+        "WebUrl": "https://some_url/some_pdf.pdf",
+        "Size": 3028,
+        "CreatedBy": {
+            "Application": {"DisplayName": "MS Graph Files", "ID": "some_id"},
+            "User": {"DisplayName": "SharePoint App"},
+        },
+        "LastModifiedBy": {
+            "Application": {"DisplayName": "MS Graph Files", "ID": "some_id"},
+            "User": {"DisplayName": "SharePoint App"},
+        },
+        "ParentReference": {
+            "DriveType": "documentLibrary",
+            "DriveId": "some_drive_id",
+            "Path": "/drive/root:/test-folder",
+            "ID": "some_id",
+        },
+        "File": {"MimeType": "image/jpeg", "Hashes": {"QuickXorHash": "quickXorHash"}},
+        "FileSystemInfo": {
+            "CreatedDateTime": "2022-12-15T12:56:27Z",
+            "LastModifiedDateTime": "2022-12-28T11:38:55Z",
+        },
+        "Image": {},
+        "Shared": {"Scope": "users"},
+        "ID": "some_id",
+    }
+}
+
+
+@pytest.mark.parametrize('client, args', UPLOAD_LARGE_FILE_COMMAND_ARGS)
+def test_upload_command_with_upload_session(mocker, client, args):
+    """
+        Given:
+            - An image to upload.
+        When:
+            - running upload new file command.
+        Then:
+            - return an result with upload session.
+     """
+    import requests
+    mocker.patch.object(demisto, 'getFilePath', return_value={'path': 'test_data/shark.jpg',
+                                                              'name': 'shark.jpg'})
+    create_upload_mock = mocker.patch.object(MsGraphClient, 'create_an_upload_session',
+                                             return_value=({"response": "", "uploadUrl": "test.com"}, "test.com"))
+    upload_query_mock = mocker.patch.object(requests, 'put', side_effect=upload_response_side_effect)
+    upload_new_file_command(client, args)
+    assert validate_upload_attachments_flow(create_upload_mock, upload_query_mock)
```

Comment:
Check here as well that the right function is called (with the session) and assert not for the regular function

Reply:
there is a test in validate_upload_attachments_flow about checking that the upload session function called 
added a test for the function without the upload session